ooniprobe/Utility/SettingsUtility.m

```diff
@@ -28,25 +29,25 @@ + (NSArray*)getSettingsForCategory:(NSString*)categoryName{
         return @[@"AppleLanguages", @"debug_logs", @"storage_usage" , @"warn_vpn_in_use"];
     }
     else if ([categoryName isEqualToString:@"test_options"]) {
-        return [TestUtility getTestOptionTypes];
-    }
-    else if ([[TestUtility getTestOptionTypes] containsObject:categoryName])
-        return [SettingsUtility getSettingsForTest:categoryName :YES];
-    else
+        return @[@"automated_testing_enabled", @"automated_testing_wifionly", @"automated_testing_charging", @"website_categories", @"max_runtime_enabled", @"max_runtime"];
+    } else {
         return nil;
+    }
 }
 
 + (NSString*)getTypeForSetting:(NSString*)setting{
-    if ([setting isEqualToString:@"experimental"] || [setting isEqualToString:@"long_running_tests_in_foreground"])
+    if ([setting isEqualToString:@"experimental"] || [setting isEqualToString:@"long_running_tests_in_foreground"]) {
         return @"bool";
-    else if ([setting isEqualToString:@"website_categories"] || [[TestUtility getTestOptionTypes] containsObject:setting])
+    } else if ([setting isEqualToString:@"website_categories"]) {
```

Comment:
What is the impact of deleting this clause?

Reply:
`[TestUtility getTestOptionTypes]` is no longer one of the expected settings type.

Reply:
ACK